modules/tinymce/src/core/test/ts/browser/init/ShadowDomEditorTest.ts

```diff
@@ -79,3 +79,31 @@ UnitTest.asynctest('Only one skin stylesheet should be loaded for multiple edito
   ]);
   Pipeline.async({}, [ Chain.asStep({}, [ nc ]) ], () => success(), failure);
 });
+
+UnitTest.asynctest('aux div should be within shadow root', (success, failure) => {
+
+  if (!ShadowDom.isSupported()) {
+    return success();
+  }
+
+  Theme();
+
+  const shadowHost = Element.fromTag('div', document);
+  Insert.append(Body.body(), shadowHost);
+  const sr = Element.fromDom(shadowHost.dom().attachShadow({ mode: 'open' }));
+  const editorDiv = Element.fromTag('div', document);
+  Insert.append(sr, editorDiv);
+
+  TinyLoader.setupFromElement((editor: Editor, onSuccess, onFailure) => {
```

Comment:
Just a thought, but given you're getting more and more tests like this, should we create a new `setupFromShadowDom` or something helper in McAgar?

Reply:
That's a great idea.

Reply:
Logged as TINY-6130